src/cheri/debug-integration.adoc

```diff
@@ -203,7 +203,15 @@ The <<m_bit>> of this capability is _only_ updated by executing <<MODESW_CAP>> o
 include::img/drootcreg.edn[]
 
 === Modified Trap-Return Instruction Behavior
-The `DRET` instruction reads the full YLEN bits of the <<mepc_y>>/<<sepc_y>> register and unseals it prior to exception return if it is a <<sentry_cap>>.
+If the platform incorporates <<section_zysentry>>,
+the `DRET` instruction reads the full YLEN bits of the <<mepc_y>>/<<sepc_y>> register and unseals it prior to exception return
+if the read capability's type
+is defined as acceptable for <<JALR_CHERI>> to unseal.
+(If there are conditions on <<JALR_CHERI>>'s willingness
```

Comment:
```suggestion
(If there are conditions on <<JALR_CHERI>>'s ability
```
willingness seems strange - it's not a teenager who we're trying to persuade to do something 😉